### DISCUSSION TITLE
Area Card | Definable if average or sum of sensor values shown

### DISCUSSION THREAD

**Discussion by Hesti85** [home-assistant/frontend#21686] [Dashboard]
Hi everyone,

at the moment the sensor values only show the average values. I term of multiple temperature sensors, this makes sense. But regarding multiple entities that measure power, it would great if you could configure to sum up the values to get the consumption of a whole area.

**Comment by ildar170975** [comment]
https://github.com/home-assistant/frontend/issues/20014
